### DISCUSSION TITLE
Show MPD/MPC now playing info in MacOS "now playing" menu bar item

### DISCUSSION THREAD

**Discussion by hgvhgv** [MusicPlayerDaemon/MPD#1920] [Q&A]
Newer versions of MacOS have a now playing menubar item (also in the control center). It picks up media playing in browsers and Apple music/Spotify (see screenshot below). Has anyone succeeded in getting MPD (directly or via mpc) show show up here? I assume succeeding would also expose mpd/mpc to the built in keyboard media controls. In the past I used https://github.com/arttuperala/kmbmpdc but it doesn't seem to work in macos 13.6.1. 

<img width="274" alt="Screenshot 2023-12-15 at 14 18 27" src="https://github.com/MusicPlayerDaemon/MPD/assets/6763386/2ff7e9d4-70b5-4fa3-bc1e-e11395a6a32a">

**Comment by Efterklang** [comment]
This command solves the problem for me.

```shell
brew install pipx
pipx install mpd-now-playable
mpd-now-playable
```
